docs/workforce-human-oidc-auth.md

```diff
@@ -0,0 +1,267 @@
+# Requirements for Client Applications implementing Workforce (Human) OIDC Auth For MongoDB
+
+# Overview
+
+## Abstract
+
+MongoDB offers OpenID Connect (OIDC) authentication and authorization for database users. OIDC auth in clients generally
+falls into one of two categories; either Workflow OIDC targeting programmatic users, which is
```

Comment:
```suggestion
falls into one of two categories: either Workflow OIDC targeting programmatic users, which is
```

Extreme nit and not really a grammar expert, but felt this should be a colon, shouldn't it?